app/assets/javascripts/sync.coffee.erb

```diff
@@ -128,7 +128,7 @@ class Sync.Pusher extends Sync.Adapter
     !!window.Pusher
 
   connect: ->
-    @client = new window.Pusher(Sync.PUSHER_API_KEY)
+    @client = <%= defined?(PusherFake) ? "new window.Pusher(#{[PusherFake.configuration.key, PusherFake.configuration.to_options].map(&:to_json).join(",")})" : "new window.Pusher(Sync.PUSHER_API_KEY)" %>
```

Comment:
Can you split this into multiple lines? We can violate DRY in order to keep this more readable.